src/devices/cpu/evolution/evo.h

```diff
@@ -0,0 +1,50 @@
+// license:BSD-3-Clause
+// copyright-holders:David Haywood
+
+#ifndef MAME_CPU_EVO_EVO_H
+#define MAME_CPU_EVO_EVO_H
+
+#pragma once
+
+enum
+{
+	EVO_PC = STATE_GENPC
+};
```

Comment:
Can you put this inside `evo_cpu_device` to keep the constants out of the global namespace?